sdk/textanalytics/Azure.AI.TextAnalytics/tests/AuthenticationTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.Testing;
+using NUnit.Framework;
+
+namespace Azure.AI.TextAnalytics.Tests
+{
+    public class AuthenticationTests : ClientTestBase
```

Comment:
I think we should either add this to Live tests or name it Live tests so it's clear.

Reply:
Fair. Once live test pr is merged, I will create a PR to move this test there, for now I will leave it here

Reply:
Comment Added to https://github.com/azure/azure-sdk-for-net/issues/9091